config/cloudinary.yml

```diff
@@ -0,0 +1,19 @@
+---
+development:
+  cloud_name: dmqyfyvkd
+  api_key: '884562686559465'
```

Comment:
generally putting keys in a repo isn't good practice but this is really just a sample app so I think this is fine.